generator.go

```diff
@@ -59,16 +60,20 @@ func NewTraceGenerator(tsender Sender, getFielder func() *Fielder, log Logger, o
 // - nspans is the number of spans in a trace.
 // If nspans is less than depth, the trace will be truncated at nspans.
 // If nspans is greater than depth, some of the children will have siblings.
-func (s *TraceGenerator) generate_spans(ctx context.Context, fielder *Fielder, level int, depth int, nspans int, timeRemaining time.Duration) {
-	if depth == 0 || nspans == 0 {
+func (s *TraceGenerator) generate_spans(ctx context.Context, fielder *Fielder, level int, depth int, nspans int, timeRemaining time.Duration, debugCount *int) {
```

Comment:
Can you explain a bit more about what debugCount is used for?

Reply:
We were having issue with loadgen not able to send giant traces. This counter was helpful to know how much spans loadgen is creating. I thought this is a useful information to have when debugging issues

Reply:
Okay, sounds useful. Could you update the func description to add that info for the param? 😄